helloworld.html

```diff
@@ -3,7 +3,6 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Bootstrap demo</title>
     <h1>Hello world</h1>
```

Comment:
[Required] The project requirements, require that you style the `h1` element with a `css` class. Please add  a `class` to the `h1` element  and include the `css stylesheet` in the `head` tag as indicated below.

![Project Screenshot from 2022-10-27 02-09-28](https://user-images.githubusercontent.com/16392046/198156134-d20f997e-3b73-467c-9ff3-7403632ec625.png)

You can achieve this as indicated in the code below
## index.html
```HTML
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Bootstrap demo</title>
   <link rel="stylesheet" href="index.css" />
</head>
    <body>
       <h1 class="favourite">Hello world</h1>
   </body>
<html>
```
## index.css
```CSS
 .favourite {
   color: blue;
}
```